src/utils/api/redditApi.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @module utils/redditApi 
+ * @desc 
+ */
+import axios from 'axios';
+
+const BASE_URL = 'https://www.reddit.com';
+
+export function searchQuery(query) {
+
```

Comment:
again, just encapsulating and leaving room to grow and extend your reddit api client. Say you want to hit another api resource on reddit, like users or subreddits, you can see it'd just be adding a new method for each. Like mentioned before you can get fancy and make this a class to mirror the endpoints of the api. Its also common to see parsing and helper methods in this same file.